jmclient/jmclient/__init__.py

```diff
@@ -20,7 +20,7 @@
                      BitcoinCoreWallet, estimate_tx_fee, WalletError,
                      create_wallet_file, SegwitWallet, Bip39Wallet)
 from .configure import (load_program_config, jm_single, get_p2pk_vbyte,
-    get_network, jm_single, get_network, validate_address, get_irc_mchannels,
+    jm_single, get_network, validate_address, get_irc_mchannels,
```

Comment:
@chris-belcher @AdamISZ removed redundant `jm_single`, that is the only change here.